src/torrentcreate.py

```
@@ -130,12 +131,12 @@ def build_mkbrr_exclude_string(root_folder, filelist):
     return exclude_str
 
 
-async def create_torrent(meta, path, output_filename, tracker_url=None, piece_size=None):
-    if piece_size is None:
+async def create_torrent(meta: dict[str, Any], path: str, output_filename: str, tracker_url: str = "", piece_size: int = 0):
+    if piece_size:
         try:
-            piece_size = meta.get('max_piece_size', None)
+            piece_size = meta.get('max_piece_size', 0)
         except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)